site/docs/components/AppDemo.tsx

```diff
@@ -0,0 +1,119 @@
+'use client';
+import { OnchainKitProvider } from '@coinbase/onchainkit';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import type { ReactNode } from 'react';
+import { http, WagmiProvider, createConfig } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { coinbaseWallet } from 'wagmi/connectors';
+import '@coinbase/onchainkit/styles.css';
+import { createWalletClient } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import type { Config } from 'wagmi';
+import { useTheme } from '../contexts/Theme.tsx';
+
+const queryClient = new QueryClient();
+
+// WARNING: This is a publicly exposed private key used only for documentation demos.
+// Do not use this key for any real transactions or store any assets in this account.
+const DUMMY_PK =
+  '0xa67bdd8a49324aa36cb3f7f7064b9b560e3aa653b774be9793415c0a6fc62cf8';
+
+const demoWalletConnector = (_config: Config) => {
+  const account = privateKeyToAccount(DUMMY_PK as `0x${string}`);
+
```

Comment:
not strictly necessary, but can do this which is kinda nice :)

```suggestion
const DUMMY_PK =
  '0xa67bdd8a49324aa36cb3f7f7064b9b560e3aa653b774be9793415c0a6fc62cf8' as const;

const demoWalletConnector = (_config: Config) => {
  const account = privateKeyToAccount(DUMMY_PK);

```

Reply:
Very nice

Reply:
BTW the reason this is slightly better is because you're allowing Typescript to check that the const adheres to `0x${string}` rather than you telling it that it adheres to `0x${string}`

In this case there'd almost never be any issues with it as we won't be updating it, but it's nice to do it the more robust way anyway